### PR TITLE
Openssl 3.0.15+quic

### PR DESCRIPTION
**Pull request by tmshort**
##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Comment by tmshort** [comment]
I was waiting for approvals...

**Comment by wbl** [comment]
Sorry for the delay, looking now

File: .github/workflows/fips-checksums.yml [hunk 1]
```diff
@@ -73,3 +73,29 @@ jobs:
         with:
           name: fips_checksum
           path: artifact/
+  verify-checksums:
```
Comment (wbl) [inline .github/workflows/fips-checksums.yml:76]: It's not clear this test is doing much for us anymore, and isn't redundant with above

**Review by wbl — APPROVED** [review]

**Review by richsalz — APPROVED** [review]
you only need watson, not me :)

File: .github/workflows/fips-checksums.yml (hunk 1 above)
Comment (tmshort) [reply .github/workflows/fips-checksums.yml:76]: It is pointless for us.

**Comment by rapsys** [comment]
Hi,

Thanks for the merge and hard work.

When will the official release tarball be available ?

Best regards

**Comment by rapsys** [comment]
Could we have the final release for it please ?

**Comment by rapsys** [comment]
@tmshort @mcollina @wbl @richsalz sorry for the tag, I just need the official tarball to release it in Mageia, best regards.

**Comment by wbl** [comment]
I think I just fixed the tag typo. Please confirm

**Comment by rapsys** [comment]
@wbl Thank's, it's released now ;) Hopefully we will get a version 3.3.x too soon ;)

**Comment by wbl** [comment]
i just released the one we've been working on. This was hard due to some major refactors. Note that we've decided to fork harder, so look at github.com/quictls/quictls. We'd appreciate your input in deciding what to do.

**Comment by richsalz** [comment]
What @wbl means, @rapsys, is that we are unlikely to do anything in the 3.3 line, and will instead be using https://github.com/quictls/quictls moving forward.

**Pull request merged**